src/color/creating_reading.js

```diff
@@ -532,13 +545,13 @@ p5.prototype.lightness = function(c) {
  * </code>
  * </div>
  *
- * <div>
+ * <div class="norender">
  * <code>
- * colorMode(RGB, 255);
+ * colorMode(RGB, 255); // Set R in RGB range to 255
  * let c = color(127, 255, 0);
- * colorMode(RGB, 1);
+ * colorMode(RGB, 1); // Set R in RGB range to 1
```

Comment:
I think this also needs to communicate that its all three (RGB), not just R. Does that make sense?
```suggestion
 * colorMode(RGB, 1); // Sets the range for red, green, and blue range to 1
```